app/src/main/java/com/capstone/nongglenonggle/presentation/view/worker/resume/main_screen/ResumeMainViewModel.kt

```
@@ -0,0 +1,129 @@
+                    birthDate = birthDate,
+                    gender = gender,
+                    certificate = certificate,
+                    certificateList = certificateList,
+                    careerList = careerList,
+                    regionList = regionList,
+                    categoryList = categoryList,
+                )
+            )
+        } catch (e: Exception) {
+
+        }
```

detekt (1.23.8)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)